code/game/objects/items/stacks/rods.dm

```diff
@@ -9,7 +9,7 @@
 	throwforce = 15.0
 	throw_speed = 5
 	throw_range = 20
-	matter = list(DEFAULT_WALL_MATERIAL = 1875)
+	matter = list(DEFAULT_WALL_MATERIAL = SHEET_MATERIAL_AMOUNT / 2)

```

Comment:
I could have sworn that there are circumstances under which BYOND will do integer division. Maybe I am just confused.